python/datasources/cdc_hiv.py

```diff
@@ -215,15 +217,25 @@ def load_atlas_df_from_data_dir(geo_level: str, breakdown: str):
                                                              usecols=lambda x: x not in cols_to_exclude,
                                                              thousands=',',
                                                              dtype=DTYPE)
-            cols_to_standard = {
-                'Cases': determinant,
-                'Rate per 100000': PER_100K_MAP[determinant],
-                'Percent': std_col.HIV_PREP_COVERAGE}
+
+            if determinant == std_col.HIV_CARE_PREFIX or determinant == std_col.PREP_PREFIX:
+                cols_to_standard = {
```

Comment:
might be more python-y to do `if determinant in [std_col.HIV_CARE_PREFIX, std_col.PREP_PREFIX'` that's just a little style thing though. I'd like to prefer that style because it's easier to add more conditions in the future